conf/environments.js

```diff
@@ -60,6 +60,18 @@ export default new Map(Object.entries({
     builtin: {
         globals: globals.es5
     },
+    es3: {
+        globals: globals.es3,
```

Comment:
emm. just found `globals` does not have `globals.es3`.  https://github.com/sindresorhus/globals/blob/main/globals.json

Reply:
oh weird, vscode gave me it as an autocomplete prompt, so i didn't check.

I'll just remove the es3 env.